src/main/java/racingcar/view/OutputView.java

```diff
@@ -0,0 +1,43 @@
+package racingcar.view;
+
+import java.util.ArrayList;
+import racingcar.model.CarData;
+
+public class OutputView {
+    public void printRoundScore(ArrayList<CarData> scoreBoard) {
+        for (CarData carScore : scoreBoard) {
+            printCarScore(carScore.getName(), carScore.getMoveDistance());
+        }
+        System.out.println();
+    }
+
+    public void printWinnerOfGame(ArrayList<String> carNames) {
+        System.out.println("최종 우승자 : " + String.join(", ", carNames));
+    }
+
+
+    public void printGameStart() {
+        System.out.println("\n실행결과");
+    }
+
+    private void printCarScore(String carName, Integer carScore) {
+
+        StringBuilder builder = new StringBuilder();
+        String dashScore = generateDashScore(carScore);
+
+        builder.append(carName);
+        builder.append(" : ");
+        builder.append(dashScore);
+
+        System.out.println(builder);
+    }
```

Comment:
```suggestion
    private void printCarScore(String carName, Integer carScore) {
        String dashScore = generateDashScore(carScore);
        System.out.printf("%s : %s\n", carName, dashScore);
    }
```
`printf` 를 활용해보시면 어떨까요? 더 깔끔하게 바꿀 수 있을 것 같아요!

Reply:
> `printf` 를 활용해보시면 어떨까요? 더 깔끔하게 바꿀 수 있을 것 같아요!

헉 ㅋㅋ 두줄로 표현이 가능하군요 배워갑니다..